x-pack/plugins/security_solution/common/endpoint/types.ts

```diff
@@ -350,7 +351,23 @@ export interface AlertEvent {
 }
 
 /**
- * The status of the host
+ * The status of the Endpoint Agent as reported by the Agent or the
+ * Security Solution app using events from Fleet.
+ */
+export enum EndpointStatus {
+  /**
+   * Agent is enrolled with Fleet
+   */
+  ENROLLED = 'enrolled',
+
+  /**
+   * Agent is unenrrolled from Fleet
+   */
+  UNENROLLED = 'unenrolled',
+}
+
+/**
+ * The status of the host, which is mapped to the Elastic Agent status in Fleet
```

Comment:
I probably missed this discussion, but it appears "status" is a bit generic and overloaded here. 

Is the field just added (the top level `status`) only going to have these states, or do we expect to add a few more states in the future to this. Like offline, errored, etc.  If we're not expecting that, and the values only concern enrollment, could the field be called `enrolled` and be a boolean? or `enrollment` and keep the values as they are.

Reply:
Is it is going to have other values e.g. installing e.t.c..